docs/GDI-SOP_github-introduction-for-maintainers.md

```diff
@@ -0,0 +1,193 @@
+5. Once you have reviewed the changes, click on the **Review changes** button, where you can choose to add a comment, approve the PR, or request changes.
+![alt text](images/GDI-SOP_github-introduction-for-maintainers_review2.png)
+
+For **more details on the reviewing process**, refer to the [**review checklist**](../docs/GDI-SOP_review-checklist.md).
+
+## General Recommendations
+
+- **Do not push directly to `main`**: Always use a PR to propose changes.
+- **Check PR origin**: Review who submitted the PR and ensure it follows repository guidelines before merging.
+- **Ensure workflow checks pass**: Before merging a PR, ensure that all GitHub Actions workflows have passed (e.g., linting).
+- **Clear commit messages**: Use descriptive commit messages to help maintain a clean history (e.g., `ZH-301 - Fix SOP typo in section 2.3`).
+
+## Workflows and Linting
+
+GitHub Workflows, also known as GitHub Actions, help **automate certain actions in the repository**. In the GDI SOP repository, workflows are used, for example, to ensure SOPs follow proper formatting and style guidelines before merging them.
+
+You can **find the workflow configurations for this repository** in the [`.github/workflows/`](../.github/workflows/) directory, and the scripts they run in the [`scripts/`](../scripts/) folder. To learn more about each specific workflow, see the description at the top of the corresponding workflow file in `.github/workflows/`.
+
+Generally speaking, if everything works as expected, you will not need to dive much into this repository's workflows. Instead, they will simply aid you with visually appealing icons at Pull Requests:
+- ✔️ - Implies that the **workflow(s) finished correctly** and all checks were passed. 
+- ❌ - Implies that **something went wrong**. This requires further investigation (see example below).
+
+Let's go over a simple example of how a workflow aided in a [recent Pull Request](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pull/38). In this PR, ``M-casado`` created a PR against the ``dev`` branch, which triggered some of the workflows:
+- Compare SOP index table ([``compare_index.yml``](../.github/workflows/compare_index.yml)). It has a ❌, letting us know that something went wrong.
+- Linter of SOPs ([``lint_sops.yml``](../.github/workflows/lint_sops.yml)). It has a ✔️, so it checked the SOPs in this branch and lets us know that they seem to be well formatted.
+
+Notice at the bottom of the image below that, once the owner of the PR noticed the error through this workflow, it was amended and a checkmark ✔️ was obtained.
+![Detecting something went wrong](images/GDI-SOP_github-introduction-for-maintainers_workflow.png)
+
+To see further we can click on the **Details** next to the workflow that failed (see image below). You can inspect for yourself the same page [**here**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/actions/runs/11477389761/job/31939356879). 
```

Comment:
This part of the description is a little advanced for someone who does not have so much technical knowledge. However, it is good that there is! For me, I need to go through it 2-3 times to understand it :-)

Reply:
I see. I may be able to elaborate a bit more - But this part of how to resolve the issues that are detected in the PRs should definitely be in the guide. It's probably complex but otherwise a newcomer may be completely lost

Reply:
Added a bit more. We'll check tomorrow if it's too confusing.